nats.go

```diff
@@ -589,21 +589,30 @@ type srv struct {
 
 // The INFO block received from the server.
 type serverInfo struct {
-	ID           string   `json:"server_id"`
-	Host         string   `json:"host"`
-	Port         uint     `json:"port"`
-	Version      string   `json:"version"`
-	AuthRequired bool     `json:"auth_required"`
-	TLSRequired  bool     `json:"tls_required"`
-	TLSAvailable bool     `json:"tls_available"`
-	Headers      bool     `json:"headers"`
-	MaxPayload   int64    `json:"max_payload"`
-	ConnectURLs  []string `json:"connect_urls,omitempty"`
-	Proto        int      `json:"proto,omitempty"`
-	CID          uint64   `json:"client_id,omitempty"`
-	ClientIP     string   `json:"client_ip,omitempty"`
-	Nonce        string   `json:"nonce,omitempty"`
-	LameDuckMode bool     `json:"ldm,omitempty"`
+	AuthRequired      bool     `json:"auth_required,omitempty"`
+	CID               uint64   `json:"client_id,omitempty"`
+	ClientConnectURLs []string `json:"connect_urls,omitempty"` // Contains URLs a client can connect to.
+	ClientIP          string   `json:"client_ip,omitempty"`
+	Cluster           string   `json:"cluster,omitempty"`
+	Dynamic           bool     `json:"cluster_dynamic,omitempty"`
+	GitCommit         string   `json:"git_commit,omitempty"`
+	GoVersion         string   `json:"go"`
+	Headers           bool     `json:"headers"`
+	Host              string   `json:"host"`
+	ID                string   `json:"server_id"`
+	IP                string   `json:"ip,omitempty"`
+	JetStream         bool     `json:"jetstream,omitempty"`
+	LameDuckMode      bool     `json:"ldm,omitempty"`
+	MaxPayload        int64    `json:"max_payload"`
+	Name              string   `json:"server_name"`
+	Nonce             string   `json:"nonce,omitempty"`
+	Port              int      `json:"port"`
+	Proto             int      `json:"proto"`
+	TLSAvailable      bool     `json:"tls_available,omitempty"`
+	TLSRequired       bool     `json:"tls_required,omitempty"`
+	TLSVerify         bool     `json:"tls_verify,omitempty"`
+	Version           string   `json:"version"`
+	WSConnectURLs     []string `json:"ws_connect_urls,omitempty"` // Contains URLs a ws client can connect to.
```

Comment:
might not need all these, i mainly updated it with the server

Reply:
I see your editor sorted them too 👎

Reply:
Hehe, can undo, there's some weird things like this client has a int64 for MaxPayload but the server reports int32, I guess we canr really change either at this point so will keep casting here differently from the server

Reply:
You can probably make it int if you want..

Reply:
I thought we wouldnt want to change this sig `func (nc *Conn) MaxPayload() int64`

Reply:
Ah yes your right..

Reply:
ok, remove JS and put this in the sme order as in the server rather than sorted

Reply:
Remove `WSConnectURLs`, it is not set by the server and instead uses `connect_urls`. I would also not have renamed `ConnectURLs` to `ClientConnectURLs` since from a client perspective, this is the connect urls :-)